priv/templates/phx.gen.live/form.ex

```diff
@@ -0,0 +1,98 @@
+defmodule <%= inspect context.web_module %>.<%= inspect Module.concat(schema.web_namespace, schema.alias) %>Live.Form do
+  use <%= inspect context.web_module %>, :live_view
+
+  alias <%= inspect context.module %>
+  alias <%= inspect schema.module %>
+
+  @impl true
+  def render(assigns) do
+    ~H"""
+    <div>
```

Comment:
Sorry, small nit. We probably don't need this `div`? It was needed before because it was a function component. We also need to decide if we want a `render` function or to define `form.html.heex`. The file would be more consistent as that's also what we use for index/show. @chrismccord, any preferences?

Reply:
@chrismccord prefers to have all of them colocated, so perhaps we do that in another PR?

Reply:
We can kill the div because it was only there for the LC which requires a root, so ✂️ away

Reply:
Done in e52c465, good catch :)

Reply:
> Colocated the templates in https://github.com/phoenixframework/phoenix/commit/e21c02560eb0cc963d1ab4d416a7a898015dcd18 :)

Love it!